keycloak/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM jboss/keycloak:2.2.1.Final
+FROM jboss/keycloak:2.3.0.Final
+
+ENV KEYCLOAK_LOGLEVEL TRACE
```

Comment:
Could this be a runtime envvar, set in Compose?  I'm guessing will want to tune the log level up/down without having to rebuild the image.